src/lando/main/models/repo.py

```diff
@@ -172,24 +172,8 @@ def _find_supporting_scm(self, pull_path) -> str:
         for scm, impl in SCM_IMPLEMENTATIONS.items():
             if impl.repo_is_supported(pull_path):
                 return scm
-        if self._is_git_repo:
-            return SCM_GIT
```

Comment:
Was this here in error originally?

Reply:
It was there to keep the HgSCM-only code from #154 work. It's no longer needed.